src/test/datascience/datascienceSurveyBanner.vscode.test.ts

```diff
@@ -60,20 +60,25 @@ import { MillisecondsInADay } from '../../client/constants';
             when(appEnv.uiKind).thenReturn(UIKind.Desktop);
             when(appEnv.channel).thenReturn(UseVSCodeNotebookEditorApi ? 'insiders' : 'stable');
             when(editorProvider.onDidOpenNotebookEditor).thenReturn(noop as any);
-            const realStateFactory = api.serviceContainer.get<IPersistentStateFactory>(IPersistentStateFactory);
-            openNotebookCountState = realStateFactory.createGlobalPersistentState<number>(
+
+            // Fake up persistant storage as this tests has been hanging while trying to update the actual mementos
```

Comment:
Very odd these test have been failing for about a week now. Seems to be hanging on the update of the PersistantState. After some poking, I just moved over to using the TestPersistentState that we already had. Felt legit to use that given how these tests work and with all the other mocks being used.